### PR TITLE
[BUG] fix doubles for FT.INFO on COORD

### LINKED ISSUE #1813: FT.INFO Unexpected memory values on _mb values
Unexpected (overflow?) memory values on xxx_mb values 

```
172.17.0.2:22525> module list
1) 1) "name"
   2) "search"
   3) "ver"
   4) (integer) 20005

172.17.0.2:22525> FT.INFO idx
21) inverted_sz_mb
22) "4.6117379832367022e+18"
23) offset_vectors_sz_mb
24) "4.6030596711026524e+18"
25) doc_table_size_mb
26) "4.592987792404906e+18"
27) key_table_size_mb
28) "4.588396850322604e+18"
29) records_per_doc_avg
30) "311.35964965820312"
31) bytes_per_record_avg
32) "6.8063759803771973"
33) offsets_per_term_avg
34) "1.1965212821960449"
35) offset_bits_per_record_avg
36) "12.198293685913086"
```

### PR DESCRIPTION
**Pull request by ashtul**
Fixes #1813

### CODE REVIEW

File: tests/pytests/common.py [hunk 1]
```diff
@@ -134,6 +134,10 @@ def skipOnExistingEnv(env):
     if 'existing' in env.env:
         env.skip()
 
+def skipOnOss(env):
```
Comment (rafie) [inline tests/pytests/common.py:137]: Did you mean `Standalone` here? Like, not an OSS cluster and not a RLEC cluster?

Comment (ashtul) [reply tests/pytests/common.py:137]: Changed to `SkipOnNonCluster`

File: tests/pytests/test_coordinator.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    conn.execute_command('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT', 'SORTABLE')
+    for i in range (100):
+        conn.execute_command('HSET', i, 't', 'Hello world!')
+    
+    idx_info = index_info(env, 'idx')
+    env.assertLess(float(idx_info['inverted_sz_mb']), 1)
+    env.assertLess(float(idx_info['offset_vectors_sz_mb']), 1)
+    env.assertLess(float(idx_info['doc_table_size_mb']), 1)
+    env.assertLess(float(idx_info['sortable_values_size_mb']), 1)
+    env.assertLess(float(idx_info['key_table_size_mb']), 1)
```
Comment (oshadmi) [inline tests/pytests/test_coordinator.py:16]: Consider also asserting greater than `0`?

**Review by oshadmi — DISMISSED** [review]

File: tests/pytests/test_coordinator.py (hunk 2 above)
Comment (ashtul) [reply tests/pytests/test_coordinator.py:16]: Nice touch

**Pull request merged**